//package core.io.protocol

import "MDropList.proto";
message ExploreData
{
	optional string starttime = 1;//探索开始时间戳
	optional string maxendtime = 2;//探索结束时间
	optional int32	totalgold = 3;//至今获得金币
	optional int32  totalexp  = 4;//至今获得经验
	optional int32  interval  = 5;//每隔interval秒刷新
	optional int32	goldonce  = 7;//每interval秒增加金币数
	optional int32	exponce	  = 8;//每interval秒增加经验
	optional bool	havedrop  = 9;//掉落奖励列表

}
message DseStageExplore
{
	optional int32 type = 1;//1请求数据 2收获 3领取掉落
	optional int32 res	= 2;//结果
	optional ExploreData info = 3;//面板数据
	optional int32 gold = 4;//获得金币
	optional int32 exp = 5;//获得经验
	optional MDropList droplist = 6;//获得的掉落
}
